### ISSUE TITLE
Adding a wildcard rewrite for both ipv4 & ipv6 breaks the ipv6 response

### ISSUE THREAD

**Issue by skerit** [AdguardTeam/AdGuardHome#3343]
### Issue Details

I tried to rewrite some specific addresses to a local ipv4 *and* an ipv6 address.
I created 2 wildcard addresses: 1 for the ipv4 and 1 for the ipv6 address, for example:

    *.kumulus.mydomain.be    192.168.50.2
    *.kumulus.mydomain.be    2222:2222:2222:2222:2222:2222:2222:2222

However, this did not exactly work.
The ipv4 response is correct, like in this example:

```
$ dig A something.kumulus.mydomain.be

; <<>> DiG 9.16.16 <<>> A something.kumulus.mydomain.be
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 39051
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;something.kumulus.mydomain.be.	IN	A

;; ANSWER SECTION:
something.kumulus.mydomain.be. 10 IN	A	192.168.50.2

;; Query time: 3 msec
;; SERVER: 192.168.50.1#53(192.168.50.1)
;; WHEN: ma jul 12 17:46:58 CEST 2021
;; MSG SIZE  rcvd: 63
```

But when I try to do an ipv6 lookup it gives a wrong response:

```
$ dig AAAA something.kumulus.mydomain.be

; <<>> DiG 9.16.16 <<>> AAAA something.kumulus.mydomain.be
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 45559
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;something.kumulus.mydomain.be.	IN	AAAA

;; Query time: 3 msec
;; SERVER: 192.168.50.1#53(192.168.50.1)
;; WHEN: ma jul 12 17:48:38 CEST 2021
;; MSG SIZE  rcvd: 47
```

As you can see: it's a NOERROR status, but the response is empty.

When I remove the ipv4 rewrite, the ipv6 response is fixed!

* **Version of AdGuard Home server:** 0.106.3
* **How did you install AdGuard Home:** GitHub release
* **How did you setup DNS configuration:** Router
* **If it's a router or IoT, please write device model:** Asus RT-AX88U
* **CPU architecture:**  AMD64
* **Operating system and version:** Ubuntu 20.04.2

### Expected Behavior

I expect it to return the ipv6 address I configured for the wildcard address, instead of returning nothing.

### Actual Behavior

The ipv4 request works and returns the ipv4 address I configured, but an ipv6 request returns an empty response.

### Additional Information

This is not a problem for non-wildcard rewrites.
So my current workaround is to add a `local.mydomain` rewrite for both ipv4 & ipv6, and then point the wildcard rewrites to that cname.

**Comment by ainar-g** [comment]
Thanks for the report!  This should be fixed as of snapshot 194ea6e on the `edge` channel.  Could you please check if the problem is fixed for you as well?

**Comment by ainar-g** [comment]
I'll close this issue for now.  Please feel free to reopen with more details if you find any issues with the fix.
